community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,119 @@
+As a community, we want to encourage diversity in all aspects, and set a standard
```

Comment:
nit: maybe remove "in all aspects"? It's not clear aspects it means, and I think "we want to encourage diversity" is sufficient on its own.

Reply:
All set. The story with this line is that it started as "we want to encourage diversity in <long list of attributes> and I reduced it to "in all aspects." But you are right, that's implied, and it's much cleaner to take out.